frontend/src/components/Header/index.js

```diff
@@ -61,7 +62,14 @@ const Header = ({ user, loadUser }) => {
           <a onClick={handleLogout}>Logout</a>
         </>
       ) : (
-        <a onClick={handleLogin}>Login</a>
+        <>
+          <a
+            href={`${environment.registerUrl}?returnUrl=${window.location}register-complete`}
```

Comment:
For scalability, I propose to have in the environment a variable called `API_URL`/`api_url` that for now will equal with `http://localhost:5001` and compose the URLs inside the app using it. The base URL is the one that will change most often / based on the environment, not each URL individually.